src/components/StaticAppBar/StaticAppBar.react.js

```diff
@@ -92,58 +164,42 @@ class StaticAppBar extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      isPopUpMenuOpen: false,
       showAdmin: false,
```

Comment:
Remove this state. It is not being set anywhere. Use isAdmin prop to check whether Admin or not.